airflow/www/views.py

```diff
@@ -1084,6 +1084,26 @@ def blocked(self):
             })
         return wwwutils.json_response(payload)
 
+    @expose('/latest_dag_runs')
+    @login_required
+    def latest_dag_runs(self):
+        """Returns the latest running DagRun for each DAG formatted for the UI. """
+        session = settings.Session()
```

Comment:
I don't think "views" should have any db access on their own and we should move away from that. Please don't open a session here and leave that to the API logic